### PR TITLE
#159631556 User can find available rooms by available filter

### PR DESCRIPTION
**Pull request by meshnesh**
#### What does this PR do?

Make the functionality where user can filter rooms by available

#### How should this be manually tested?

* Run the application.
* Open the find room tab, click the availability tab then select available to get the available rooms.

#### Any background context you want to provide?

* Before the rooms could not be filtered by available rooms, the filter did not work

#### What are the relevant pivotal tracker stories?
[#159631556](https://www.pivotaltracker.com/story/show/159631556)

#### Screenshots (if appropriate)

ezgif com-video-to-gif

### CODE REVIEW

**Comment by TheDancerCodes** [comment]
Hey @meshnesh 
I have just reviewed you PR and it seems that the drop down hasn't been updated to match the Invision Mockup.
The drop down still has `Unavailable` instead of `All Rooms`

File: app/src/main/java/com/andela/mrm/find_rooms/DropdownFilterAdapter.java [hunk 1]
```diff
@@ -87,6 +91,22 @@ public void setValues(final int position) {
                 @Override
                 public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
                     if (isChecked) {
+                        if (texts.get(position).equals("Available Rooms")) {
```
Comment (idrisadetunmbi) [inline app/src/main/java/com/andela/mrm/find_rooms/DropdownFilterAdapter.java]: 👎 I am not sure this is ever appropriate. What is this meant to do? restart  the activity?!

Comment (meshnesh) [reply app/src/main/java/com/andela/mrm/find_rooms/DropdownFilterAdapter.java]: @idrisadetunmbi It checks if the selected dropdown is `Available Rooms` then it restarts the `FindRoomActivity`

Comment (idrisadetunmbi) [reply app/src/main/java/com/andela/mrm/find_rooms/DropdownFilterAdapter.java]: Either way, I don't think it is the right approach.
1. Restarting the activity does not have any effect in relation to the feature you are working on.
2. The intent flag combinations you are using will rearrange the [back stack](https://developer.android.com/guide/topics/fundamentals/tasks-and-back-stack.html) of the app in a way, and I don't think that is a desirable effect.

You might want to rethink your approach to what you are trying to do.

**Pull request closed without merging**